ci-operator/jobs/openshift/os/openshift-os-master-presubmits.yaml

```diff
@@ -175,6 +175,242 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )rhcos-9-build-test-qemu,?($|\s.*)
+  - agent: kubernetes
```

Comment:
since you have separated commits for 96 and c10s, can you move the 96 definition here to the other commit?